README.rst

```diff
@@ -69,7 +70,10 @@ Emcache has currently support, among many of them, for the following commands:
 - **delete** The command allows for explicit deletion of items.
 - **touch** The command is used to update the expiration time of an existing item without fetching it.
 - **increment/decrement** Commands are used to change data for some item in-place, incrementing or decrementing it.
-
+- **cache_memlimit** This command allow set in runtime cache memory limit.
+- **stats** Show a list of required statistics about the server, depending on the arguments.
+- **verbosity** Command control STDOUT/STDERR info, choose level and look logging memcached.
+- **quit** Soft close connection with server.
```

Comment:
I would prefer to release something with what we have in main right now, so I would appreciate if we can remove the `quit` command, since the PR is not yet merged

Also I would like to limit the PR to bugs or important changes. But we can discuss this later.